.github/workflows/levm_bench.yaml

```diff
@@ -0,0 +1,47 @@
+name: LEVM benchmarks
```

Comment:
you also want https://github.com/lambdaclass/lambda_ethereum_rust/commit/0a93d977a890ae4404f80bc3f70636073cda3dfc#diff-fbedb504135d1fa140a0a84c8f428de49626a6b7a2ee68a17da5d941474b22a8

Reply:
[57ea30a](https://github.com/lambdaclass/lambda_ethereum_rust/pull/869/commits/57ea30a5b2f410f5640f9c6227ce9d8273f14e8d)

Reply:
EF tests will be added later so we don't want that workflow right now